spec/features/instructors/students/index_spec.rb

```diff
@@ -14,4 +14,28 @@
     expect(page).to have_content(bob.name)
   end
 
+  it 'links to viewing a student' do
+    hans = Instructor.create!(name: "Hans", subject: "skiing", teaches_children: true, years_experience: 30)
+    gretchen = hans.students.create!(name: 'Gretchen', age: 20, subject: "cross-country skiing", returning_student: true, level:"advanced")
+    tristan = hans.students.create!(name: 'Tristan', age: 14, subject: "snowboarding", returning_student: false, level:"advanced")
+    bob = hans.students.create!(name: 'Bob', age: 21, subject: "skiing", returning_student: false, level:"advanced")
+
+    visit "/instructors/#{hans.id}/students"
+    click_button "View Gretchen"
+
+    expect(current_path).to eq("/students/#{gretchen.id}")
+  end
+
+  it 'links to editing a student' do
```

Comment:
So I put my test in the "edit_spec.rb" Do you have a spec file called that? Wondering now which was is correct.